std_msgs/String mapName
float32[] baseLLA
std_msgs/Int8 systemMode
std_msgs/Int8 systemSignal
std_msgs/Int8 systemState
std_msgs/Int8 systemHealth
std_msgs/Int8 lapCount
std_msgs/Int8 kiapiSignal
std_msgs/Int8 headingSet